.travis.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
not sure we need both travis and circle. they should be capable of doing the same things.